index.js

```diff
@@ -115,7 +116,7 @@ ${errors.map((line, i) => `${i + 1}. ${line}`).join('\n')}`);
 			try {
 				result = compile(preprocessOption ? (await preprocess(file.contents.toString(), preprocessOption)).toString() : file.contents.toString(), {
 					filename: file.path,
-					...options
+					...omit(options, ['preprocess'])
```

Comment:
Omitted because the svelte compiler was throwing an error for the existence of the 'preprocess' entry in the options.